src/neo-vm/ExecutionEngine.cs

```diff
@@ -1154,6 +1194,84 @@ private bool ExecuteInstruction()
             return true;
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        private bool ExecuteTry(int catchOffset, int finallyOffset)
+        {
+            if (catchOffset < 0 || finallyOffset < 0) return false;
```

Comment:
Can we ensure that the compiler always put the catch and the finally after the try?

Reply:
I think we should ensure. @lightszero What do you think?

Reply:
In devpack-dotnet,It is for sure.
but I think we should remove this.

Reply:
done